Replacing the info email for the info GitHub repo (#456)
pyladies/pyladies: www/_posts/2014/2014-04-26-Attend-Google-IO-with-PyLadies.md

* Award recipients must register within 2 weeks of receiving award letter and signup process

> All information here will not be made public, and will be reviewed by PyLadies leadership only.  Please [open an issue](https://github.com/pyladies/info/issues/new) regarding any questions you have.

Reviewer: ```suggestion
All information here will not be made public, and will be reviewed by PyLadies leadership only.  Please [contact us](https://github.com/pyladies/info/issues/new) regarding any questions you have.
```

Reviewer 2: How are we going to enforce that? Issues are public on the repo. I offhand do not know about make issues private on a public repo. See: https://www.bountysource.com/issues/52815258-confidential-private-issues-on-public-repo.

Reviewer: I think the part where it says "will not be made public, and will be reviewed by PyLadies leadership only." refers to the application to the 2014 Google IO lottery. This text appears in the 2014 blog post https://www.pyladies.com/blog/Attend-Google-IO-with-PyLadies/Attend-Google-IO-with-PyLadies/ at the bottom of the post is a google forms.
The forms is closed.

So my understanding is this is not about making the GitHub issues private, but about the submitted form (which is now defunct)
Maybe we should just remove the "Please contact us" sentence 🤔

Reviewer 2: Ahhh I see. Yes, let's change language to `this is no longer available`?

Author: I think we can leave _as is_ since it is an old blog post, and probably no one is going to try to apply to it anymore. If we decide to change the wording, we can do it on a later PR. I just wanted to make sure that the info email wasn't available in case anyone was _really_ trying to find it.
